src/scripts/snabb_doc.sh

```diff
@@ -75,8 +75,8 @@ function index_out { echo $docdir/index.html;   }
 function build_doc1 {
     ((cd $(repo_path)/src && git checkout $1 && make doc/snabbswitch.html) \
         && mv $(repo_path)/src/doc/snabbswitch.html $2) \
- || ((cd $(repo_path)/src && git checkout $1 && make obj/doc/snabbswitch.html) \
-        && mv $(repo_path)/src/obj/doc/snabbswitch.html $2)
+ || ((cd $(repo_path)/src && git checkout $1 && make obj/doc/snabb.html) \
+        && mv $(repo_path)/src/obj/doc/snabb.html $2)
```

Comment:
This doesn't work

1. for this PR because the old version of SnabbDoc is run
2. for other PRs because they might be behind `next`

See #856 for a fix targeting `next`. I think merging that PR should make the CI pass.